src/components/MainContent/index.tsx

```diff
@@ -170,22 +215,20 @@ export const MainContent = ({ target }: MainContentProps) => {
         </Section>
       </Section>
 
-      <Section name="accessibility">
-        <MainHeader text="Accessibility Concerns" />
+      <Section name={names.accessibility}>
+        <MainHeader text={names.accessibility} />
         <Passage>{accessibility}</Passage>
       </Section>
 
-      <Section name="evidence">
-        <MainHeader text="Evidence Required" />
+      <Section name={names.evidence}>
+        <MainHeader text={names.evidence} />
         <NumberList>{renderListItems(target.evidence)}</NumberList>
       </Section>
 
-      <Section name="tasks">
-        {RenderTaskModels(target.taskModels, target.stem, target.evidence, target.rubrics)}
-      </Section>
+      {RenderTaskModels(target.taskModels, target.stem, target.evidence, target.rubrics, subNames)}
```

Comment:
Destructure these variables at the top of the scope.

Reply:
The only issue I have with destructureing these is that many of the destructured names are the same as other names that are used. To desctructure them, we have to change the names which would be pretty similar to how they are already named.

Reply:
Yeah, at this point it's probably not worth it to refactor but it may be worth it to make the names different at some point just do its a bit more succinct.